src/sentry/plugins/sentry_webhooks/plugin.py

```diff
@@ -103,7 +104,7 @@ def send_webhook(self, url, payload):
             verify_ssl=False,
         )
 
-    def notify_users(self, group, event, fail_silently=False):
-        payload = self.get_group_data(group, event)
+    def notify_users(self, group, event, triggering_rules, fail_silently=False):
```

Comment:
Just to future proof (and what I'm going to check in other plugins) could we add kwargs support?

```suggestion
    def notify_users(self, group, event, triggering_rules, fail_silently=False, **kwargs):
```